sdk/attestation/azure-security-attestation/src/private/crypto/inc/crypto.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
```

Comment:
Please make this fix to `main` and merge it here, otherwise I think it will be living only in the `feature/amqp` branch for a long time.

Reply:
This change was reverted in the current codebase. We can do a copyright string cleanup pass in the future if it's important.